source/_integrations/template.markdown

```diff
@@ -164,6 +164,10 @@ binary_sensor:
           description: The attribute and corresponding template.
           required: true
           type: template
+      friendly_name: 
```

Comment:
This change is not correct.

`friendly_name` is for old-style (platform) formatted templates.
It has been replaced by `name` for new style templates.